src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css

```diff
@@ -17,6 +17,10 @@ text-cell-component .notebook-preview {
 	user-select: text;
 }
 
+.notebook-preview .rangeHighlight {
+	background-color: rgba(253, 255, 0, 0.2);
```

Comment:
where did this color come from?

Reply:
I copied this from vscdoe editor styles defined in editorColorRegistry.ts. They have this for find highlight decoration. I prob need to add for dark theme as well

Reply:
Please make sure you address this and follow established patterns from vscode